app/src/main/java/com/example/harvestfresh/fragments/CartFragment.java

```diff
@@ -85,9 +102,16 @@ public void onClick(View v) {
         });
     }
 
+    public void getTotal() {
```

Comment:
nit: naming
We typically in java use methods that start with "get" to return a value, so this naming would suggest returning the total for use elsewhere, while it doesn't return anything. it would be better to name this similar to what the method is doing, like "updateTotalTextView" or "refreshTotalText". The more accurate naming helps people in the future understand what they can expect from your API without having to open the reference and read the whole thing.

Typical conventions I see:
getX() - gets the value of X
setX(x) - sets the value of X
isX() - gets a boolean value, like "isTotalOverOneHundred" or something.